vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -735,21 +735,6 @@ def allowed_snapshots(_options = {})
     result
   end
 
-  def allowed_datastore_storage_controller(_options = {})
-    # TODO: NetAppFiler is an ActAsArModel and doesn't support arel syntax yet
-    NetAppFiler.all.collect(&:name).index_by { |n| n }
-  end
-
-  def allowed_datastore_aggregate(_options = {})
-    result = {}
-    controller = get_value(@values[:new_datastore_storage_controller])
-    return result if controller.blank?
-    naf = NetAppFiler.find_by_name(controller)
-    naf_type = get_value(@values[:new_datastore_fs_type]) == "NFS" ? :aggregates : :volumes
-    naf.send(naf_type).each { |a| result[a] = a }
-    result
-  end
```

Comment:
Yes! In addition to returning an empty hash, we should log a deprecation warning or something so we can eventually remove them.